.github/workflows/ci.yml

```diff
@@ -11,23 +11,41 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  elixir:
-    uses: ./.github/workflows/_elixir.yml
-  rust:
-    uses: ./.github/workflows/_rust.yml
-    secrets: inherit
+  # These workflows are build-only and don't have tests, so the
+  # benefit to running them in CD in addition to the PR and
+  # merge group workflows is negligible.
   kotlin:
+    if: github.workflow == "Continuous Integration"
     uses: ./.github/workflows/_kotlin.yml
     secrets: inherit
   swift:
+    if: github.workflow == "Continuous Integration"
     uses: ./.github/workflows/_swift.yml
     secrets: inherit
+
+  # We build tauri artifacts in CD, no need to build it before here.
+  tauri:
+    if: github.workflow == "Continuous Integration"
+    uses: ./.github/workflows/_tauri.yml
+    secrets: inherit
+
+  # These workflows have tests which may catch semantic
+  # merge errors resulting from the final merge to main
+  # that the PR and merge group workflows don't catch.
```

Comment:
How? The merge group should prevent semantic conflicts.

Reply:
Yeah good point, I thought so, but it was late when I was thinking through this and wasn't 100% sure.

I'll update all of these.